src/App.jsx

```diff
@@ -14,45 +16,120 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
+const SORT_DIRECTION_REVERSED = 'reversed';
+let CHANGED_DIRECTION = false;
+let CHANGED_SORTING = false;
+
+function getPreparedGoods(goods, { sortField, direction }) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    CHANGED_SORTING = true;
+
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_NAME:
+          return good1.localeCompare(good2);
+
+        case SORT_FIELD_LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (direction === SORT_DIRECTION_REVERSED) {
```

Comment:
```suggestion
  if (isReversed) {
```
Just create a boolean as you can have only two values (reversed/not reversed)